modin/experimental/core/execution/native/implementations/hdk_on_native/test/test_dataframe.py

```diff
@@ -2117,6 +2122,10 @@ def sort(df, ascending, **kwargs):
             ascending=ascending,
         )
 
+    @pytest.mark.skipif(
+        hdk_version == "0.7.0",
+        reason="https://github.com/modin-project/modin/issues/6514",
```

Comment:
I don't quite get this. We specifed this issue here to skip the test case but the PR resolved the the issue and it was closed. Could you elaborate the case here?

Reply:
The PR fixes the test failure by skipping the test case for HDK 0.7.0. This test case does not fail on the HDK built from the master branch. After updating to the next HDK version, this test case will not be skipped.